x/ibc/02-client/types/querier.go

```diff
@@ -0,0 +1,38 @@
+package types
+
+// query routes supported by the IBC client Querier
+const (
+	QueryClientState    = "clientState"
```

Comment:
These should be camel case, see `distr` and other modules for examples.

Reply:
`distribution` uses snake_case, while `slashing` uses camelCase. Which one should I use then? cc @alexanderbez